File: server/src/main/kotlin/com/pidulgi/server/member/dto/response/MemberGetMeResponse.kt
package com.pidulgi.server.member.dto.response

import com.pidulgi.server.member.entity.type.Gender

data class MemberGetMeResponse(

    val memberId: Long,
    val publicImages: List<MemberImageResponse> = emptyList(),
    val privateImages: List<MemberImageResponse> = emptyList(),
    val nickname: String,
    val gender: Gender,
    val age: Int,
    val birthYear: Int,
    val bio: String?,
    val likes: Long,
)

File: app/app/Model/Member/Response/MemberGetMeResponse.swift
struct MemberGetMeResponse: Codable {

    let memberId: Int64
    let publicImages: [MemberImageResponse]
    let privateImages: [MemberImageResponse]
    let nickname: String
    let gender: String
    let age: Int
    let birthYear: Int
    let bio: String?
    let likes: Int
}
